templates/zerver/help/roles-and-permissions.md

```diff
@@ -32,7 +37,3 @@ Cloud. There are two kinds of guest for billing purposes:
   to change over time. For example, single-stream guests can currently see
   the list of members in the organization, but we may remove that ability in
   the future.
-
-## Related articles
-
-* [Stream permissions](/help/stream-permissions)
```

Comment:
Hmm, I think it might be better to leave this link here, even though it's linked above, for scannability?

Reply:
Why is that link any more notable than any other link in the body?  Should we be pulling out all cross-links in the body into "Related articles" on each page?

Reply:
I don't think we need or want to do that as a systematic rule, but where it's particularly important information in understanding something, which is the case here, it seems worth listing in both places.